apps/storybook/src/mockData/clineMessages.ts

```diff
@@ -0,0 +1,694 @@
+import seedrandom from "seedrandom"
```

Comment:
is this really necessary? Would regular Math.random give such another view, or hardcoded random numbers?